lib/daru/vector.rb

```diff
@@ -879,41 +879,67 @@ def to_s
       to_html
     end
 
-    # Create a summary of the Vector using Report Builder.
-    def summary(method=:to_text)
-      ReportBuilder.new(no_title: true).add(self).send(method)
+    # Create a summary of the Vector
+    # @params [Fixnum] indent_level
+    # @return [String] String containing the summary of the Vector
+    # @example
+    #   dv = Daru::Vector.new [1, 2, 3]
+    #   puts dv.summary
+    #
+    #   =
+    #     n :3
+    #     n valid:0
+    #     median: 2
+    #     mean: 2.0000
+    #     std.dev.: 1.0000
+    #     std.err.: 0.5774
+    #     skew: 0.0000
+    #     kurtosis: -2.3333
+    def summary(indent_level=0)
+      nval = count_values(*Daru::MISSING_VALUES)
+      summary = '  =' * indent_level + "= #{name}" \
+                "\n  n :#{size}" \
+                "\n  n valid:#{nval}"
```

Comment:
Well I actually followed [this](https://github.com/SciRuby/daru/pull/305/files#diff-8129d70abcb202c11af77e9269873527L891). Changing it.